src/foam/u2/wizard/CompositeWAO.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * @license
+ * Copyright 2022 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.wizard',
+  name: 'CompositeWAO',
```

Comment:
I wonder if we should add composite: true, to Interfaces to make the generation of composites automatic, like proxies, clients and skeletons.